### ISSUE TITLE
[SampleCard-ExpenseReport; Config-Default]: Screen reader does not announce updated/changed 'un-selected' state for 'Reject' button, instead stays silent.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7375]
https://microsoft.visualstudio.com/OS/_workitems/edit/37986783
